.travis.yml

```diff
@@ -1,4 +1,9 @@
 language: generic
+env:
+  - DB_USERNAME='locksmith_test'
```

Comment:
this runs the test suite in parallel, setting each env variable for one of the builds, so another solution needs to be found to set the env variables all at once in each build

Reply:
my research suggests using `global:` as a sub-thingy of env should do what you're trying to do